airbyte-integrations/connectors/source-dockerhub/acceptance-test-config.yml

```diff
@@ -21,4 +21,8 @@ tests:
       configured_catalog_path: "integration_tests/configured_catalog.json"
       # testing sequentially for same results can fail because of pull counts increasing for an image between runs
       ignored_fields:
-        "docker_hub": ["pull_count", "last_updated"]
+        docker_hub:
+          - name: pull_count
+            bypass_reason: "fast changing data"
+          - name: last_updated
+            bypass_reason: "fast changing data"
```

Comment:
This seems to imply we are testing against a real docker image that someone's using?